src/material/datepicker/multi-year-view.ts

```diff
@@ -39,6 +39,39 @@ export const yearsPerPage = 24;
 
 export const yearsPerRow = 4;
 
+/** Gets remainder that is non-negative, even if first number is negative */
+function euclideanModulo (a: number, b: number): number {
+  return (a % b + b) % b;
+}
+
+/** We pick a "starting" year such that either the maximum year would be at the end */
+/* or the minimum year would be at the begining of a page. */
```

Comment:
It seems like your function header comments are a bit strangely formatted. I would expect this:
```ts
/** We pick a "starting" year such that either the maximum year would be at the end
 * or the minimum year would be at the beginning of a page. */
```
or preferably this
```ts
/** 
 * We pick a "starting" year such that either the maximum year would be at the end
 * or the minimum year would be at the beginning of a page. 
 */
```
"begining" spelled wrong as well.

Reply:
Thanks for spotting that!